packages/node/src/integrations/tracing/express.ts

```diff
@@ -135,8 +135,11 @@ export function expressErrorHandler(options?: {
  * Setup an error handler for Express.
  * The error handler must be before any other middleware and after all controllers.
  */
-export function setupExpressErrorHandler(app: { use: (middleware: ExpressMiddleware) => unknown }): void {
-  app.use(expressErrorHandler());
+export function setupExpressErrorHandler(
+  app: { use: (middleware: ExpressMiddleware) => unknown },
+  options?: Parameters<typeof expressErrorHandler>[0],
```

Comment:
l/m: As a consumer of libraries I always find it immensly annoying when libraries define types like this. Imo we should define an `ExpressErrorHandlerOptions` interface and use that in both places.